src/app/public/modules/numeric/numeric.pipe.ts

```diff
@@ -25,7 +25,7 @@ export class SkyNumericPipe implements PipeTransform {
     private readonly skyNumeric: SkyNumericService
   ) { }
 
-  public transform(value: number, config: any): string {
+  public transform(value: number, config: any = {}): string {
```

Comment:
Makes `config` optional, so that template-checking doesn't fail when only the value is provided `{{ value : skyNumeric }}`.